File: app/src/main/java/me/wcy/music/utils/ToastUtils.java
package me.wcy.music.utils;

import android.content.Context;
import android.widget.Toast;

/**
 * Toast工具类
 * Created by wcy on 2015/12/26.
 */
public class ToastUtils {
    private static Context sContext;

    public static void setContext(Context context) {
        sContext = context.getApplicationContext();
    }

    public static void show(int resId) {
        Toast.makeText(sContext, resId, Toast.LENGTH_SHORT).show();
    }

    public static void show(String text) {
        Toast.makeText(sContext, text, Toast.LENGTH_SHORT).show();
    }
}

File: app/src/main/java/me/wcy/music/utils/ToastUtils.kt
package me.wcy.music.utils

import android.content.Context
import android.widget.Toast

/**
 * Toast工具类
 * Created by wcy on 2015/12/26.
 */
object ToastUtils {
    private var sContext: Context? = null
    private var sToast: Toast? = null
    fun init(context: Context?) {
        sContext = context!!.applicationContext
    }

    fun show(resId: Int) {
        show(sContext!!.getString(resId))
    }

    fun show(text: String?) {
        if (sToast == null) {
            sToast = Toast.makeText(sContext, text, Toast.LENGTH_SHORT)
        } else {
            sToast!!.setText(text)
        }
        sToast!!.show()
    }
}